iosApp/iosApp/Pages/NearbyTransit/NearbyTransitView.swift

```diff
@@ -118,21 +118,22 @@ struct NearbyTransitView: View {
                 ScrollView {
                     LazyVStack(spacing: 0) {
                         ForEach(transit, id: \.id) { nearbyTransit in
-                            switch onEnum(of: nearbyTransit) {
+                            let nearby: StopsAssociated = nearbyTransit
```

Comment:
This was because I had some unrelated compilation error within the case statement (it was the `pushNavEntry` arg thing I mentioned above), and instead of giving a useful error, it was instead a type error about `nearbyTransit` not being a `StopsAssociated`, so I added this to get more useful compilation errors in the future.